content/v3/repos/releases.md

```diff
@@ -57,7 +57,13 @@ View the latest published release for the repository.
 
 ## Get a release by tag name
 
-Get a release with the specified tag. Users must have push access to the repository to view draft releases.
+Get a release for the specified git tag.
+
+{{#tip}}
+
+Releases that are drafts usually don't have their git tag created yet, so they will have to be looked up by listing all releases for the repository. Only users with push access will receive listings for draft releases.
```

Comment:
It may be good to clarify the "usually" part. My understanding is that the draft release wouldn't be available through the tag API if the tag hasn't been created yet. But if the tag itself was created already, *then* a draft release... I'm not sure, would be curious to check.